docs/reference/connectors/http.md

```diff
@@ -276,3 +161,24 @@ Response metadata takes the following general form:
 
 Setting the `$correlation` metadata on an outbound request will result in the response
 being tagged with the `$correlation` value set in the corresponding request.
+
+
+## Example implementation
+
+This is a relatively basic client server system that replays JSON formatted lines of data from a text file over HTTP to a server. The
```

Comment:
It is very unclear what `This` is referring to here, as nothing is there, except a small mermaid graph and a link somewhere. This section should be removed.